app/views/layouts/_flash_message.html.erb

```diff
@@ -2,4 +2,5 @@
   <div class='alert alert-<%= key %>' >
     <%= value %>
     </div>
-<% end %>
\ No newline at end of file
+<% end %>
+<% flash.clear %>
```

Comment:
By default, Rails clears flash messages once they are displayed.

Is there anything that wasn't cleared by the code above?

Reply:
I found `/users/me` was not clearing errors. I think this is because they only gets cleared on a redirect, and it doesn't redirect. Let me know if there a better way to handle this.

https://github.com/procore/Procore-Sample-ROR/blob/129277b550a6d3bbcaf986878bd596383da30111/app/controllers/users_controller.rb#L12-L14